pages/Documents/ConversationalAI/Third-PartyBots/hook-configuration.md

```diff
@@ -186,7 +204,22 @@ Example Payloads:
 }
 ```
 
-###### Lex
+###### Minimal working Faas example as code
+
+```js
+const { payload } = input;
+
+if (payload.message) {
+  input.payload.message += 'preHook';
+}
```

Comment:
I know it's annoying, but could you change it from "input.payload.message" to "payload.message" on this example functions too?